internal/endpoints/metrics.go

```diff
@@ -35,10 +55,29 @@ func incRequests() {
 	requests.With(p.Labels{}).Inc()
 }
 
+// IncConsumedMessages increments the message count by 1
+func IncConsumedMessages() {
```

Comment:
I suspect these public methods were added so that you could reach the endpoints package data from within the kafka package?  If those metrics are moved into the kafka package, then these methods are no longer needed.